diff -Naur bygfoot-2.2.1-source/configure.in bygfoot-2.2.1-source.tpg/configure.in
--- bygfoot-2.2.1-source/configure.in	2008-03-02 16:23:22.000000000 +0000
+++ bygfoot-2.2.1-source.tpg/configure.in	2008-06-28 13:37:28.000000000 +0000
@@ -56,9 +56,9 @@
 
 if test x$ENABLE_GST = xyes; then
 	HAVE_GSTREAMER=0
-	dnl start with 0.8
-	GST_MAJORMINOR=0.8
-	GSTREAMER_REQUIRED=0.8
+	dnl start with 0.10
+	GST_MAJORMINOR=0.10
+	GSTREAMER_REQUIRED=0.10
 	PKG_CHECK_MODULES(GST, \
 		gstreamer-$GST_MAJORMINOR >= $GSTREAMER_REQUIRED,
 		HAVE_GSTREAMER=1,HAVE_GSTREAMER=0)
